src/AddTodoForm.js

```diff
@@ -1,12 +1,21 @@
 import React from "react";
 
-const AddTodoForm = () => {
+const AddTodoForm = (props) => {
+
+
+    const handleAddTodo = (event) => {
+        event.preventDefault(event);
+        const todoTitle = event.target.title.value;
+        props.onAddTodo (todoTitle);
+
```

Comment:
we may want to add 

`event.target.title.value = "";`

to clear value in input after we submit the form